src/pointer-analysis/value_set_analysis.h

```diff
@@ -14,32 +14,80 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #define USE_DEPRECATED_STATIC_ANALYSIS_H
 #include <analyses/static_analysis.h>
+#include <util/xml_expr.h>
+#include <util/xml.h>
 
 #include "value_set_domain.h"
 #include "value_sets.h"
+#include "value_set.h"
 
 class xmlt;
 
-class value_set_analysist:
+template<class DomainValueSet>
```

Comment:
Still think looks a bit awkward to have this non-conforming to name convention (pointing out in case this isn't intentional, not to start a flame war :stuck_out_tongue:)